CONTRIBUTING.md

```diff
@@ -68,10 +78,87 @@ If you have questions about the project, using PyScript, or anything else, pleas
 ```
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **NOTE**: To access your local build paste `http://localhost:8080` into your browser
 
+
+Now that node and npm have both been updated `make setup` should work, and you can continue [setting up your local environment](#setting-up-your-local-environment) without problems (hopefully).
+
+### Developing
+
+* First, make sure you are using the latest version of the pyscript main branch
+
+```
+    git pull upstream main
+```
+
+* Update your fork with the latest changes
+
+```
+    git push origin main
+```
+
+* Activate the conda environment (this environment will contain all the necessary dependencies)
+
+```
+    conda activate pyscriptjs/env/
+```
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **NOTE**: We are assuming you are in the root folder. If you are in the pyscriptjs you can run `conda activate env/` instead.
```

Comment:
Is that how it goes if you follow the `make setup` instructions?
It creates an  environment named `env`?

Reply:
Yeah, I think its because the `environment.yml` file doesn't contain `name: pyscript`? 🤔